src/constants.js

```diff
@@ -1,3 +1,11 @@
 'use strict'
 
 export const DEFAULT_API_PORT = 8080
+
+export const MUTUAL_TLS_AUTH_TYPE = 'mutual-tls-auth'
+
+export const BASIC_AUTH_TYPE = 'basic-auth'
+
+export const CUSTOM_TOKEN_AUTH_TYPE = 'custom-token-auth'
+
+export const JWT_TOKEN_AUTH_TYPE = 'jwt-token-auth'
```

Comment:
@bradsawadye technically you've just said `JSON Web Token token auth`... So you can just leave it at `jwt-auth`

Reply:
addressed in https://github.com/jembi/openhim-core-js/pull/1082/commits/ec811918f9e9bfb287e433282d3c8b526563de66